sdk/src/rent.rs

```diff
@@ -67,23 +74,43 @@ mod tests {
 
     #[test]
     fn test_due() {
-        let rent = Rent::default();
+        let rent_with_base_zero = Rent::default();
```

Comment:
when we change the default rent to a non-zero value, this won't read right....

Reply:
this code is designed to work whether the default rent is zero or non-zero.  to turn on rent, just change 
```
DEFAULT_LAMPORTS_PER_BYTE_YEAR

Reply:
have more clear name now

Reply:
@ParthDesai: the first test will fail if you change DEFAULT_LAMPORTS_PER_BYTE_YEAR, now....

Reply:
@rob-solana my bad. Changed all ref to DEFAULT_LAMPORTS_PER_BYTE_YEAR